packages/api/src/external/commonwell/document/document-query.ts

```diff
@@ -327,6 +330,16 @@ export async function internalGetDocuments({
         },
         log,
       });
+
+      await tallyDocQueryProgress({
+        patient: { id: patient.id, cxId: patient.cxId },
+        progress: {
+          errors: cwErrs.length,
+        },
+        type: "download",
+        requestId,
+        source: MedicalDataSource.COMMONWELL,
+      });
```

Comment:
[Context](https://metriport.slack.com/archives/C0616FCPAKZ/p1727123487795029?thread_ts=1727122368.356919&cid=C0616FCPAKZ)

we dont add failed commonwell queries to our error download tally

Reply:
Is it an issue to call `tallyDocQueryProgress` here before `initPatientDocQuery` has been called?

Reply:
Well yea the tally would be off then. We only mark the total of documents that are returned from this function

Reply:
To be clear this is failing when querying for doc refs? Or when we try to download the document once we have retreived the Doc ref